elasticsearch/src/main/java/com/amazon/opendistroforelasticsearch/sql/elasticsearch/storage/script/filter/lucene/TermQuery.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *    Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License").
+ *    You may not use this file except in compliance with the License.
+ *    A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    or in the "license" file accompanying this file. This file is distributed
+ *    on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *    express or implied. See the License for the specific language governing
+ *    permissions and limitations under the License.
+ *
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.elasticsearch.storage.script.filter.lucene;
+
+import static com.amazon.opendistroforelasticsearch.sql.elasticsearch.data.type.ElasticsearchDataType.ES_TEXT_KEYWORD;
+
+import com.amazon.opendistroforelasticsearch.sql.data.model.ExprValue;
+import com.amazon.opendistroforelasticsearch.sql.data.type.ExprType;
+import org.elasticsearch.index.query.QueryBuilder;
+import org.elasticsearch.index.query.QueryBuilders;
+
+/**
+ * Lucene query that build term query for equality comparison.
+ */
+public class TermQuery extends LuceneQuery {
+
+  @Override
+  protected QueryBuilder doBuild(String fieldName, ExprType fieldType, ExprValue literal) {
+    if (fieldType == ES_TEXT_KEYWORD) { // Assume inner field name is always "keyword"
```

Comment:
could you add doc for this limitation?

Reply:
Good idea! Added a `limitation` folder with a single doc for now. Copy-pasted the up-to-date content from ODFE doc. Will make it doctest with example in explain API PR. Thanks!

1. MySQL reference: https://dev.mysql.com/doc/mysql-reslimits-excerpt/8.0/en/
2. ODFE doc: https://opendistro.github.io/for-elasticsearch-docs/docs/sql/limitation/
3. New limitation doc: https://github.com/dai-chen/sql/blob/lucene-pushdown-optimization/docs/user/limitations/limitations.rst